lib/vector2d.rb

```diff
@@ -1,5 +1,19 @@
 # encoding: utf-8
 
+require 'contracts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.